File: Beecrowd/1 - Beginner/1101 - 1190/1133 - Rest of a Division/1133_RestofaDivision.py
value1 = int(input())
value2 = int(input())

if value1 > value2:
    value1, value2 = value2, value1

for i in range(value1 + 1, value2):
    if i % 5 == 2 or i % 5 == 3:
        print(i)

File: Beecrowd/1 - Beginner/1101 - 1190/1133 - Rest of a Division/1133_RestofaDivision.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int value1, value2;
    cin >> value1 >> value2;

    if (value1 > value2) swap(value1, value2);

    for (int i = value1 + 1; i < value2; i++) {
        if (i % 5 == 2 || i % 5 == 3) {
            cout << i << endl;
        }
    }
    
    return 0;
}